### ISSUE TITLE
No warning for field from default tuple value in conditional

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#32575]
No warning reported for `x.Item2.ToString()`:
```C#
class Program
{
    static void F<T, U>(bool b, T t, U u)
        where U : class
    {
        var x = b ? (t, u) : default;
        x.Item1.ToString(); // warning: maybe null
        x.Item2.ToString();
    }
}
```

**Comment by jcouv** [comment]
It looks like the behavior changed since this bug was filed. We currently (June 17th) report no warning at all, which is surprising given that `T` is unconstrained.
